manifests/server.pp

```diff
@@ -60,12 +60,19 @@
 
   validate_bool($directoryenvs)
 
+  $service = $servertype ? {
+    'passenger'    => 'httpd',
```

Comment:
it'd probably be better to put this in params.pp, because the name of the apache service differs across Linux distros (httpd vs apache2) and probably in other platforms as well. (no idea what it gets called under *BSD)